src/nodes/victron-virtual.js

```diff
@@ -800,7 +800,8 @@ module.exports = function (RED) {
                 }[v] || 'unknown'),
                 persist: false
               },
-              { name: 'Settings/ValidTypes', type: 'i', value: 0x7 }
+              { name: 'Settings/ValidTypes', type: 'i', value: 0x7 },
+              { name: 'Settings/ShowUIControl', type: 'i', value: 1, persist: true }
```

Comment:
To confirm my understanding: This makes a setting `ShowUIControl` available, but this setting is not configurable through the Node-RED interface, correct?

I'm approving, assuming that this is the desired behavior :)

Reply:
That is correct.